Packs/DeCYFIR/Integrations/decyfir/decyfir.py

```diff
@@ -0,0 +1,483 @@
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+
+    def say_hello(self, name):
+        return f'Hello {name}'
+
+    def get_severity(self, risk_score: int):
+        if risk_score > 8:
+            return IncidentSeverity.CRITICAL
+        elif risk_score > 5:
+            return IncidentSeverity.HIGH
+        elif risk_score >= 3:
+            return IncidentSeverity.MEDIUM
+        elif risk_score >= 1:
+            return IncidentSeverity.LOW
+        else:
+            return IncidentSeverity.UNKNOWN
+
+    def request_decyfir_api(self, endpoint) -> str:
+        response = requests.get(f"{endpoint}")
+        if response.status_code == 200:
+            if len(response.text) > 0:
+                return json.dumps(response.json())
+        else:
+            print(f"Error: =>  {response.status_code} error with API request")
+            return ""
+
+    def get_decyfir_data(self, after_val: int, is_first_fetch: bool, org_api_key: str, incident_type: str) -> str:
+        endpoint_env = f"{QA_API_PATH}"
+
+        size: int = MAX_INCIDENTS_TO_FETCH
+        if is_first_fetch:
+            size = 1000
+
+        api_query = "&" + f"key={org_api_key}&" + f"size={size}&" + f"after={after_val}"
+
+        return_data: json = {}
+        for cat_type in VAR_ATTACK_SURFACES_SUB_TYPES:
+            endpoint = f"{endpoint_env}" + f"/{VAR_ATTACK_SURFACE}?" + f"type={cat_type}" + api_query
+            data = self.request_decyfir_api(endpoint)
+            return_data[cat_type] = data
+
+        for cat_type in VAR_IMPERSONATION_AND_INFRINGEMENT_SUB_TYPE:
+            endpoint = f"{endpoint_env}" + f"/{VAR_IMPERSONATION_AND_INFRINGEMENT}?" + f"type={cat_type}" + api_query
+            data = self.request_decyfir_api(endpoint)
+            return_data[cat_type] = data
+
+        for cat_type in VAR_DATA_BREACH_AND_WEB_MONITORING_SUB_TYPES:
+            endpoint = f"{endpoint_env}" + f"/{VAR_DATA_BREACH_AND_WEB_MONITORING}?" + f"type={cat_type}" + api_query
+            data = self.request_decyfir_api(endpoint)
+            return_data[cat_type] = data
+
+        for cat_type in VAR_SOCIAL_AND_PUBLIC_EXPOSURE_SUB_TYPES:
+            endpoint = f"{endpoint_env}" + f"/{VAR_SOCIAL_AND_PUBLIC_EXPOSURE}?" + f"type={cat_type}" + api_query
+            data = self.request_decyfir_api(endpoint)
+            return_data[cat_type] = data
+
+        return json.dumps(return_data)
+
+    def prepare_incident_json(self, source_brand: str, alert_type: str, alert_subtype: str, name: str, date_val: str,
+                              severity: int, details: str, record_id: str) -> Dict[str, Any]:
+
+        incident_owner = "Administrator"
+        return_data = {
+            "type": "" + f"{alert_type}",
+            "name": name,
+            "occurred": dateparser.parse(date_val).strftime(DATE_FORMAT),
+            "owner": incident_owner,
```

Comment:
I think we do not support adding owner as part of creation of incident as when investigated, the owner is set randomly. Have you checked this?